### PR TITLE
Update the global hub 2.9 document

### PR DESCRIPTION
**Pull request by yanmxa**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ You can read about the use cases for multicluster global hub in [Use Cases](./gl
 
 ### Multicluster Global Hub Operator
 
-The Multicluster Global Hub Operator contains the components of multicluster global hub. The Operator deploys all of the required components for global multicluster management. The components include `multicluster-global-hub-manager` in the global hub cluster and `multicluster-global-hub-agent` in the managed hub clusters.
+The Multicluster Global Hub Operator contains the components of multicluster global hub. The Operator deploys all of the required components for global multicluster management. The components include `multicluster-global-hub-manager` and `multicluster-global-hub-grafana` in the global hub cluster and `multicluster-global-hub-agent` in the managed hub clusters.
```
Comment (clyang82) [inline doc/README.md:44]: Do we need to mention postgres and kafka?

File: doc/README.md [hunk 2]
```diff
@@ -294,18 +312,23 @@ After installing the global hub operand, the global hub manager starts running a
 
 #### Local compliance status sync job
 
-  At 0 o'clock every day, based on the policy status and events collected by the manager on the previous day. Running the job to summarize the compliance status and change frequency of the policy on the cluster, and store them to the `history.local_compliance` table as the data source of grafana dashboards. Please refer to [here](./how_global_hub_works.md) for more details.
+  At 0 o'clock every day, based on the policy status and events collected by the manager on the previous day. Running the job to summarize the compliance status and change frequency of the policy on the cluster, and store them to the `history.local_compliance` table as the data source of grafana dashboards. Even if the job runs once a day, it can also run multiple times within the day without changing the result. Please refer to [here](./how_global_hub_works.md) for more details.
```
Comment (clyang82) [inline doc/README.md:315]: do not understand `Even if the job runs once a day, it can also run multiple times within the day without changing the result.`

Comment (yanmxa) [reply doc/README.md:315]: That means if we run the job several times in one day. the result will be the same.

File: doc/README.md (hunk 1 above)
Comment (yanmxa) [reply doc/README.md]: Yes. Since the operator will do such a thing, let's add this information.

Done!

File: doc/README.md (hunk 2 above)
Comment (yanmxa) [reply doc/README.md:315]: @clyang82 Maybe we can remove it from here?

Comment (clyang82) [reply doc/README.md:315]: yes. just remove to avoid some confusions.

Comment (yanmxa) [reply doc/README.md]: @clyang82 Done.

**Pull request merged**